src/Administration/Resources/config/services.xml

```diff
@@ -155,11 +146,7 @@
         <service id="Shopware\Administration\Framework\Routing\KnownIps\KnownIpsCollector"/>
 
         <service id="Shopware\Administration\Notification\NotificationDefinition">
```

Comment:
I guess, we still need our special annotation syntax as comment somewhere here, otherwise it might be overseen :grin: